### ISSUE TITLE
DLL load failed while importing custom_rasterizer_kernel: The specified procedure could not be found.

### ISSUE THREAD

**Issue by Akhilesh-2024** [comfyanonymous/ComfyUI#9961]
Think issue is with the wheel that comes from Hunyuan3DWrapper wheel which is not working i installed the correct one still same issue can anyone tell which one should be installing 

D:\ComfyUI\ComfyUI_windows_portable\python_embeded>python.exe -V 
Python 3.12.8 

D:\ComfyUI\ComfyUI_windows_portable\python_embeded>python.exe -m pip list 
Package Version ------------------------- ------------ accelerate 1.10.1 aiofiles 24.1.0 aiohappyeyeballs 2.4.6 aiohttp 3.11.12 aiohttp_socks 0.10.1 aiosignal 1.3.2 albucore 0.0.24 albumentations 2.0.8 annotated-types 0.7.0 attrs 25.1.0 av 14.1.0 beautifulsoup4 4.13.5 certifi 2025.1.31 cffi 2.0.0 chardet 5.2.0 charset-normalizer 3.4.1 click 8.3.0 colorama 0.4.6 colour-science 0.4.6 cryptography 46.0.1 dataclasses-json 0.6.7 Deprecated 1.2.18 diffusers 0.35.1 easydict 1.13 einops 0.8.1 filelock 3.17.0 frozenlist 1.5.0 fsspec 2025.2.0 gdown 5.2.0 gitdb 4.0.12 GitPython 3.1.45 h11 0.16.0 h2 4.3.0 hpack 4.1.0 huggingface-hub 0.35.0 hyperframe 6.1.0 idna 3.10 imageio 2.37.0 importlib_metadata 8.7.0 Jinja2 3.1.5 joblib 1.5.2 jsonschema 4.25.1 jsonschema-specifications 2025.9.1 kornia 0.8.0 kornia_rs 0.1.8 lazy_loader 0.4 llvmlite 0.45.0 markdown-it-py 4.0.0 MarkupSafe 3.0.2 marshmallow 3.26.1 matrix-nio 0.25.2 mdurl 0.1.2 mpmath 1.3.0 msvc_runtime 14.44.35112 multidict 6.1.0 mypy_extensions 1.1.0 networkx 3.4.2 numba 0.62.0 numpy 2.2.3 opencv-python 4.12.0.88 opencv-python-headless 4.12.0.88 packaging 24.2 pillow 11.1.0 pip 25.0.1 pixeloe 0.1.4 platformdirs 4.4.0 pooch 1.8.2 propcache 0.3.0 psutil 7.0.0 pybind11 3.0.1 pycparser 2.22 pycryptodome 3.23.0 pydantic 2.11.9 pydantic_core 2.33.2 pygit2 1.17.0 PyGithub 2.8.1 pygltflib 1.16.5 Pygments 2.19.2 PyJWT 2.10.1 PyMatting 1.1.14 pymeshlab 2025.7 PyNaCl 1.6.0 PySocks 1.7.1 python-socks 2.7.2 PyYAML 6.0.2 referencing 0.36.2 regex 2024.11.6 rembg 2.0.67 requests 2.32.3 rich 14.1.0 rpds-py 0.27.1 safetensors 0.5.2 scikit-image 0.25.2 scikit-learn 1.7.2 scipy 1.15.2 sentencepiece 0.2.0 setuptools 75.8.0 shellingham 1.5.4 simsimd 6.5.3 smmap 5.0.2 soundfile 0.13.1 soupsieve 2.8 spandrel 0.4.1 stringzilla 4.0.13 sympy 1.13.1 threadpoolctl 3.6.0 tifffile 2025.9.9 timm 1.0.19 tokenizers 0.21.0 toml 0.10.2 torch 2.6.0+cu126 torchaudio 2.6.0+cu126 torchsde 0.2.6 torchvision 0.21.0+cu126 tqdm 4.67.1 trampoline 0.1.2 transformers 4.49.0 transparent-background 1.3.4 trimesh 4.8.2 typer 0.18.0 typing_extensions 4.12.2 typing-inspect 0.9.0 typing-inspection 0.4.1 unpaddedbase64 2.1.0 urllib3 2.3.0 uv 0.8.19 wget 3.2 wrapt 1.17.3 xatlas 0.0.11 yarl 1.18.3 zipp 3.23.0 

D:\ComfyUI\ComfyUI_windows_portable\python_embeded>python.exe -c "import torch; print(torch.version.cuda); print(torch.cuda.is_available())" 
12.6 True

**Comment by comfyui-wiki** [comment]
Check this: https://comfyui-wiki.com/en/tutorial/advanced/3d/huanyuan3d-2#12-texture-generation-component-installation
